plugin/keyring_vault/keyring_vault-test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+ADD_EXECUTABLE(keyring_vault-test
```

Comment:
This program is not a test suite, so it should not work with ctest or test ... it is just mine application to check if keyring API is working with multiple threads. Maybe it is only confusing user, should I remove it from source tree?